gateway/templates-initiate/default.conf.template

```diff
@@ -0,0 +1,9 @@
+server {
+    listen [::]:80;
+    listen 80;
+    server_name $DOMAIN;
+    location ~/.well-known/acme-challenge {
```

Comment:
Why don't we just incorporate this rule into the default image? Then we end up with a single `docker-compose.yml` and include something like [this](https://gist.github.com/kizzx2/782b500a81ce46b889903b1f80353f21) to restart the certbot container every day?